README.md

```diff
@@ -233,3 +233,16 @@ Split a cell horizontally, where `t` is a value ranged 0 to 1 representing where
 ### updateNormals(quads, cell)
 
 Updates all of the normals of a cell to be that of the computed normal of the cell.
+
+### fromFunction(f, nu, nv, computeNormals = true, uPeriodic = false, vPeriodic = false)
+
+Given a function `f(u, v)` (`(u, v) ∈ [0, 1] × [0, 1]`) that returns an n-dimensional vector as a length-n Array, this function creates a `nu × nv` grid of quads. The normals are computed and normalized using the cross product across grid lines unless either `computeNormals` is `false` or the comtpued vectors are not three-dimensional. The returned quads have `cells`, `positions`, and, when appropriate, `normals`. If normals are computed, `uPeriodic` and `vPeriodic` determines whether the differences used in the cross product are one-sided (the default) or wrap around periodically in the corresponding direction.
```

Comment:
And then the number of true/false booleans kinda blew up, so I thought maybe best not to refine too much depending on its uncertain future.